### ISSUE TITLE
provisiond: integration test against real BCDB actor

### ISSUE THREAD

**Issue by zaibon** [threefoldtech/zos#334]
We need to have a full test suite of test against the [provision actor](https://github.com/threefoldtech/jumpscaleX_threebot/blob/development/ThreeBotPackages/threefold/tfgrid_workloads/actors/workload_manager.py)

~~Blocked on https://github.com/threefoldtech/jumpscaleX_threebot/issues/90~~

**Comment by serboctor** [comment]
https://github.com/threefoldtech/jumpscaleX_threebot/issues/90 is fixed.

**Comment by muhamadazmy** [comment]
@zaibon I wanted to start on this, but currently there are some major changes happening on the development_bcdb branch. Should we put this on hold, or start with what we have?

**Comment by zaibon** [comment]
@muhamadazmy no point working on this if the interface of gedis/bcdb will change soon. We need a stable version to test against.
